src/ola_flesche_ex/ex04/walker.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Ola Flesche Hellenes'
+__email__ = 'olhellen@nmbu.no'
+import random
+
+
+class Walker:
+    def __init__(self, x0, h):
+        self.x0 = x0
+        self.h = h
+        self.steps = 0
+
+    def move(self):
+        self.steps += 1
+        step = 2*random.randint(0, 1)-1
+        self.x0 = self.x0 + step
+
+    def is_at_home(self):
+        if self.x0 == self.h:
+            return True
```

Comment:
you need to return false otherwise, or it may cause problems for other applications